client/src/main/java/tds/assessment/Item.java

```diff
@@ -97,6 +98,17 @@ public void setGroupKey(String groupKey) {
     }
 
     /**
+     * @return the block id of the item group
```

Comment:
This is the note form the Admin Test Package specification:

"Used to group items into blocks for the field test selection algorithm.
Note that the Test Authoring assigns "A" for all items."